content/docs/contributing/policy.md

```diff
@@ -3,59 +3,109 @@ title: Feature Policy
+We love to receive both feature requests and PRs which add stuff to cert-manager; the community is at the heart of what we do!
 
-# Features we will not allow
+If you're thinking of adding a feature, we recommend you read this doc to maximize the chances of your contribution getting the attention it deserves and hopefully to get it merged quickly!
 
-Certain features have been requested before, but have not been implemented properly or not able to be implemented without breaking the security model.
+We recommend creating an issue first for it to be discussed with the cert-manager maintainers. Another possibility is bringing it up in a community meeting discussion for an open discussion with the maintainers and community on the implementation.
 
-Below is a list of a few of these:
+## Feature Sizing: Getting Your Change Accepted
 
-## Vendoring Kubernetes related APIs outside of the `k8s.io/` namespace
+We evaluate new features and PRs based on their size; either they're small or large.
+
+### Smaller Features
+
+Many contributions are small. That usually - but not always - means that implementing them won't require many lines of code to be added or changed, and in any case they should be easy
+for maintainers to review. A PR being small is a good thing; if you can down-scope your feature to make it smaller, we won't complain!
+
+If you believe your feature is small, please feel free to just raise a PR and optionally also post a link to your PR in the [cert-manager-dev slack channel](./README.md#slack). Usually a sufficiently small PR can be merged without too much ceremony. If we think it's actually a larger piece of work, we'll let you know.
+
+### Larger Features
+
+If you're not sure whether your PR is small, or if you know it's bigger, you'll want to speak to us first before raising a PR. This
+will help to ensure that your PR is something we're likely to merge to avoid wasting your time. It'll also make it easier
+for us to do the design process.
+
+Larger PRs should normally come with a design document. This doesn't need to be a huge piece of work; rather, it should specify
+_why_ we'd want a given feature, _how_ it'll be implemented, and perhaps evaluate pros and cons. You can add this design in a
+GitHub [feature request](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=feature-request.md) or [bug report](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=bug.md).
+
+We may also have a video call or a discussion about a design document, with you included, so that we can plan how a feature should
+look. It'll be pretty informal and casual; we just want to make sure we're all on the same page!
```

Comment:
Thought: upstream SIGs seem to discuss designs in their dev meetings and it looks like usually a person can just add a design to be discussed to the meeting notes (seems like usually after having had the PR with the design open for a little while to give maintainers a chance to take a look). Would it make sense for us to also do that?

Reply:
Yeah I'll call adding things to meeting notes as an option explicitly; nice suggestion, thanks :+1: